backend/src/Request/Admin/AdminProfile/AdminProfileUpdateBySuperAdminRequest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Request\Admin\AdminProfile;
+
+use App\Entity\ImageEntity;
+
+class AdminProfileUpdateBySuperAdminRequest
+{
+    private int $id;
+
+    private string $name;
+
+    /**
+     * @var string|null
+     */
+    private $phone;
+
+    /**
+     * @var string|null
+     */
+    private $imagePath;
```

Comment:
why this ?

Reply:
For storing the image path which are provided by the user in order to update the image of the admin profile. It is different from image filed which is set for storing image entity.